### DISCUSSION TITLE
Display only root folder structure in pnp search web part

### DISCUSSION THREAD

**Discussion by sarbani04** [microsoft-search/pnp-modern-search#3576] [Q&A] [answered]
Hello Team,

We want to display root folder contents of a particular folder in document library.
We have used below query template to display the structure of "Folder A" but it is displaying all the folders of each sub-folders of "Folder A".
Please help us to resolve the issue.

ParentLink:"https://sharepoint-tenant.sharepoint.com/sites/sitename/document_library/Folder A" AND ContentType=Folder

**Comment by wobba** [comment, marked as answer]
Have you tried `ParentLink="https://sharepoint-tenant.sharepoint.com/sites/sitename/document_library/Folder A"`, using = (equals) and not : (contains)?

**Reply by sarbani04** [reply]
Hi Wobba, 

Thanks, that worked!

**Reply by sarbani04** [reply]
Hi, 
This seemed to have worked initially, but today when I checked again.. it is not working. Could you suggest any other fix for this?

**Reply by wobba** [reply]
Seems KQL parsing for some reason don't check : vs =. Add this to the Refinement field instead as FQL seems to work: `ParentLink:equals("https://tenant.sharepoint.com/teams/asite/Shared Documents/a/b")`
